inst/templates/vscode-c_cpp_properties.json

```diff
@@ -0,0 +1,21 @@
+{
+  "configurations": [
+    {
+      "name": "Mac",
+      "includePath": [
+        "${workspaceFolder}/**",{{{ linking_to_includes }}}
+        "/Library/Frameworks/R.framework/Resources/include",
+        "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include",
+        "/usr/local/include"
+      ],
+      "macFrameworkPath": [
+        "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks"
+      ],
+      "compilerPath": "/usr/bin/clang",
+      "cStandard": "c99",
+      "cppStandard": "c++11",
+      "intelliSenseMode": "macos-clang-x64"
```

Comment:
I _think_ this comes automatically with the compiler path so it probably doesn't need to be set.

I noticed that sometimes when I change the compiler in the actual UI for this properties file, if I change from clang->gcc it gives me a note that it automatically updated the intelliSenseMode

Reply:
I haven't set this on purpose, but it came from one of my actual config files. So yeah I guess something I did in the UI set it. I can remove and see if this tends to eventually get added by VS Code.